src/main/java/modify_event_use_case/ModifyEventController.java

```diff
@@ -0,0 +1,29 @@
+package modify_event_use_case;
+
+/**
+ * Controller Class. Will be used by the event modification screen to execute modification. Calls methods
+ * from the input boundary.
+ * @author Daniel Livshits
+ */
+public class ModifyEventController {
+    private final ModifyEventInputBoundary inputBoundary;
+
+    public ModifyEventController(ModifyEventInputBoundary inputBound){
+        this.inputBoundary = inputBound;
+    }
+
+    /**
+     * Calls the input boundary to execute the necessary steps event modification.
```

Comment:
Capitalization of the first word after the parameter name and @return. This applies to all the javadocs.